torch/csrc/jit/codegen/cuda/kernel.cpp

```diff
@@ -339,7 +339,10 @@ size_t numberOfReductionSegments(
       (z_block ? 1 : grid_dim.z);
 }
 
-std::array<size_t, 2> gridReductionTempBufferSizes(CudaKernel* entry) {
+std::array<size_t, 2> gridReductionTempBufferSizes(
+    CudaKernel* entry,
```

Comment:
Can we drop `entry` as it doesn't seem to be used anymore?

Reply:
We still need it for traversing graph few lines down.

Reply:
Ah, my bad. I'd suggest to pass just `fusion` or `fusion->exprs(true)`.